rest/oidc_test_provider.go

```diff
@@ -13,28 +13,37 @@ import (
 	"crypto/rsa"
 	"crypto/x509"
 	"encoding/base64"
+	"encoding/json"
 	"fmt"
 	"net/http"
 	"net/url"
+	"strconv"
 	"strings"
 	"text/template"
 	"time"
 
-	"strconv"
-
-	"github.com/coreos/go-oidc/jose"
-	"github.com/coreos/go-oidc/key"
 	"github.com/couchbase/sync_gateway/auth"
 	"github.com/couchbase/sync_gateway/base"
+	"gopkg.in/square/go-jose.v2"
```

Comment:
We'll need this mirroring into couchbasedeps and including in the manifest, as it's a new dependency.

Reply:
Thank you Ben for pointing this out. The go-jose repo has been mirrored and updated to latest revision by running the Jenkins job: http://server.jenkins.couchbase.com/job/update-couchbasedep/84